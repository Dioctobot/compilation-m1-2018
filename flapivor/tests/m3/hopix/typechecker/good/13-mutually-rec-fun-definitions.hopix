def double_omega : Int -> Int (x) =
    omega (x)
and omega : Int -> Int (x) =
    double_omega (x)

val main : Int = double_omega (73)