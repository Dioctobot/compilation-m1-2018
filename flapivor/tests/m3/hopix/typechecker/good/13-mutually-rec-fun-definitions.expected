double_omega : Int -> Int
omega : Int -> Int
main : Int
